core/domain/auth_services_test.py

```
@@ -268,6 +270,70 @@ def test_get_all_external_accounts(self) -> None:
                 auth_services.get_all_external_accounts(), mock_accounts
             )
 
+    def test_create_external_account(self) -> None:
+        input_account = auth_domain.ExternalAccount()
+        output_account = auth_domain.ExternalAccount('aid', 'a@a.com', False)
```

Ruff (0.14.14)

[error] Boolean positional value in function call

(FBT003)